install/0000_90_cluster-version-operator_02_servicemonitor.yaml

```diff
@@ -84,6 +88,7 @@ spec:
         cluster_operator_up{job="cluster-version-operator"} == 0
       for: 10m
       labels:
+        namespace: openshift-cluster-version
```

Comment:
`ClusterOperatorDown` already has `namespace` in Telemetry (this time via `cluster_operator_up`), so I'd rather drop this line from the commit as redundant.

We could consider a `max by (namespace, name, ...) (...)` aggregation, if we wanted to exclude `pod` to avoid alert churn whenever the CVO pod cycled.  But we're not all that consistent on this front today, so if we do decide to do this sort of thing, we should probably put in enough thought to make the change consistently (e.g. I'm not sure why we are including `endpoint` but not `service` in `ClusterNotUpgradeable` today).

Reply:
Thank you, fixed in https://github.com/openshift/cluster-version-operator/pull/800/commits/d525eb97a2c971b4837210963d86d3b617452891.

I have also added the aggregation operator `max by` to `CannotRetrieveUpdates`, `ClusterOperatorDown`, `ClusterOperatorDegraded`, and `ClusterOperatorFlapping` to avoid the alert churn whenever the CVO pod cycles in the new commit https://github.com/openshift/cluster-version-operator/pull/800/commits/2b8ef2a4c653df59cf67991b5ecc8c827767fe91.

I am not sure of the purpose of the `endpoint` or potentially `service` in `ClusterNotUpgradeable`, so I have not made this modification.